Mathlib/Analysis/LocallyConvex/Polar.lean

```diff
@@ -150,6 +150,35 @@ theorem polar_univ (h : SeparatingRight B) : B.polar Set.univ = {(0 : F)} := by
     _ = ε := mul_one _
 #align linear_map.polar_univ LinearMap.polar_univ
 
+theorem polar_subMulAction (m : SubMulAction 𝕜 E) : B.polar m = { y | ∀ x ∈ m, B x y = 0 } := by
```

Comment:
I think that we should use `{S : Type*} [SetLike S E] [SubMulActionClass S 𝕜 E] (m : S)` so that the lemma automatically applies to `Submodule`s.

Reply:
You mean `SMulMemClass`?

Reply:
Yes, sorry for confusion.

Reply:
Thanks for the suggestion!